### PR TITLE
New badges

### PR DESCRIPTION
**Pull request by kelson42**

### CODE REVIEW

File: .github/workflows/CI.yml
Comment (Jaifroid) [inline .github/workflows/CI.yml:4]: Thanks @kelson42, I think it's a good idea to name the file CI.yml, but maybe this descriptive name could be "Continuous Integration" instead of just "CI" which might be cryptic to new contributors exploring the code.

File: README.md
Comment (Jaifroid) [inline README.md:17]: Thanks for adding this. Looks good!

File: .github/workflows/CI.yml
Comment (kelson42) [reply .github/workflows/CI.yml:4]: @Jaifroid I just have tried to make it everywhere the same. It is mostly a matter of standardisation. CI is pretty common in the software industry, I believe this shoudl be known, and this is explained on Wikipedia https://en.wikipedia.org/wiki/CI. The naming does not matter much to me, but considering that this has implication on the on the badges as well, I believe the shorted/easier, the better.

**Comment by Jaifroid** [comment]
Thanks for this PR @kelson42 . It looks good, with a possible change of descriptive name for the CI YAML file (see comment in code).

I would prefer to merge this after we have merged #686, as we are hoping to get that finalized today or tomorrow so we can finally release v3.1 using GitHub Actions. We could have released manually by now, but this seemed like a good moment to set up and test CI with GitHub Actions, given that Travis is broken.

File: .github/workflows/CI.yml
Comment (Jaifroid) [reply .github/workflows/CI.yml:4]: OK! That sounds like a good reason!

**Review by mossroy — APPROVED** [review]

**Review by Jaifroid — APPROVED** [review]
I need to do some manipulation to clean up master, then I can rebase this PR's branch to get rid of the conflict before we can merge.

**Comment by Jaifroid** [comment]
@kelson42 I've now rebased this branch/PR on latest master. Let us know if it is ready to be squashed/merged, or else press the button yourself! Thanks for this PR.

**Pull request merged**
